### ISSUE TITLE
Jmeter Tests Looping 

### ISSUE THREAD

**Issue by garyc0221** [Azure-Samples/jmeter-aci-terraform#63]
Hello,

I see that there is another issue that is talking about there test looping although I don't know if its similar.

I have a test plan that is hitting a certain site with the thread group a follows:
```
<ThreadGroup guiclass="ThreadGroupGui" testclass="ThreadGroup" testname="Thread Group" enabled="true">
        <stringProp name="ThreadGroup.on_sample_error">continue</stringProp>
        <elementProp name="ThreadGroup.main_controller" elementType="LoopController" guiclass="LoopControlPanel" testclass="LoopController" enabled="true">
          <boolProp name="LoopController.continue_forever">false</boolProp>
          <intProp name="LoopController.loops">-1</intProp>
        </elementProp>
        <stringProp name="ThreadGroup.num_threads">1000</stringProp>
        <stringProp name="ThreadGroup.ramp_time">5</stringProp>
        <boolProp name="ThreadGroup.scheduler">true</boolProp>
        <stringProp name="ThreadGroup.duration">300</stringProp>
        <stringProp name="ThreadGroup.delay">0</stringProp>
        <boolProp name="ThreadGroup.same_user_on_next_iteration">false</boolProp>
      </ThreadGroup>
```
Now the problem is once the test is finished it never stops. It is on a constant loop despite the duration being set to 5 minutes. I can tell the test has stopped via azures application insights. 
When I stop the worker and controller container the test will stop.

Any help would be appreciated thanks.
